app/javascript/__tests__/modules/listingDetailsAside/__snapshots__/ListingDetailsLottery.test.tsx.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ListingDetailsLottery display if lottery is complete 1`] = `
+<div
+  className="border-b pt-4 text-center"
+>
+  <h4
+    className="undefined undefined"
```

Comment:
are these `undefined` classes doing anything?

Reply:
I think this is happening on the Bloom side 🤷‍♀️  I can take a look, but for the short term shouldn't cause a visual issue